web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserEndpoint.java

```diff
@@ -16,28 +15,19 @@
 import java.util.List;
 
 /**
- * Implementation of {@link UserController} that receives and serves SOAP messages.
+ * Implementation of {@link UserEndpoint} that receives and serves SOAP messages.
  */
 @WebService(targetNamespace = SoapWebServiceConstants.DEFAULT_NAMESPACE + "user")
-public class SoapUserController implements UserController {
+public class SoapUserEndpoint implements UserEndpoint {
 
-    private static final Logger logger = LogManager.getLogger(SoapUserController.class.getSimpleName());
-
-    private final TokenRegistry registry;
-
-    /**
-     * Creates a new {@code SoapUserController}.
-     */
-    public SoapUserController() {
-        registry = new HashTokenRegistry();
-    }
+    private static final Logger logger = LogManager.getLogger(SoapUserEndpoint.class.getSimpleName());
 
     @Override
     public String login(@WebParam(name = "username") String username, @WebParam(name = "password") String password) {
         logger.debug("Soap request for login (username='{}', password='{}').", username, password);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2068)